code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -1,4 +1,5 @@
 #define C2NAMEREAGENT	"[initial(reagent.name)] (Has Side-Effects)"
```

Comment:
This isn't used anywhere, you can probably remove it.

Reply:
Removed